.github/workflows/coverage-linux-without-intl.yml

```
@@ -0,0 +1,53 @@
+name: coverage-linux-without-intl
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - 'deps/**'
+      - 'benchmark/**'
+      - 'tools/**'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'doc/**'
+      - 'deps/**'
+      - 'benchmark/**'
+      - 'tools/**'
```

```suggestion
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths-ignore:
      - '**.md'
      - 'benchmark/**'
      - 'deps/**'
      - 'doc/**'
      - .github/**
      - '!.github/workflows/coverage-linux-without-intl.yml'
  push:
    branches:
      - master
      - main
    paths-ignore:
      - '**.md'
      - 'benchmark/**'
      - 'deps/**'
      - 'doc/**'
      - .github/**
      - '!.github/workflows/coverage-linux-without-intl.yml'
```
Like https://github.com/nodejs/node/blob/master/.github/workflows/coverage-linux.yml.